jinja_utils_test.py

```diff
@@ -108,3 +119,17 @@ def test_evaluate_object(self):
         parsed_dict = jinja_utils.evaluate_object(orig_dict, {'b': 'c'})
         self.assertEqual(orig_dict, {'a': '{{b}}'})
         self.assertEqual(parsed_dict, {'a': 'c'})
+
+        # int type input used.
+        parsed_object = jinja_utils.evaluate_object(34, {})
+        self.assertEqual(parsed_object, 34)
+
+    def test__log2_floor_filter(self):
+        # disabling pylint warning for protected access
+        # pylint: disable=protected-access
```

Comment:
Put this line at the top of `test__log2_floor_filter` and put the enable line at the end of the function as well.
1) You don't want to disable protected-access for the entire file
2) Helps with readability.

Reply:
Added.
Thanks